Corona-Warn-App/src/main/res/layout/trace_location_organizer_trace_locations_item.xml

```diff
@@ -0,0 +1,106 @@
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintEnd_toStartOf="@+id/menu_action"
+        app:layout_constraintStart_toEndOf="@+id/icon"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Jahrestreffen der deutschen SAP Anwendergruppe" />
+
+    <TextView
+        android:id="@+id/address"
+        style="@style/body2"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="16dp"
+        android:textColor="@color/colorTextPrimary2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/icon"
+        app:layout_constraintTop_toBottomOf="@id/description"
+        tools:text="Hauptstr. 3, 69115 Heidelberg Hauptstr. 3, 69115 Heidelberg Hauptstr. 3, 69115 Heidelberg" />
+
+    <TextView
+        android:id="@+id/duration"
+        style="@style/body2"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/icon"
+        app:layout_constraintTop_toBottomOf="@id/address"
+        app:layout_constraintVertical_bias="0.0"
+        tools:text="21.01.21 18:00 - 22.02.21 21:00 Uhr 21.01.21 18:00 - 22.02.21 21:00 Uhr" />
+
+    <ImageButton
+        android:id="@+id/menu_action"
+        style="@style/CardOverFlowButton"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.Barrier
+        android:id="@+id/button_barrier"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:barrierDirection="bottom"
+        app:constraint_referenced_ids="address,duration,icon" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/checkin_action"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="12dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="8dp"
+        android:text="@string/trace_location_organizer_list_item_action_checkin"
+        android:textColor="@color/colorAccent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/button_barrier"
+        app:strokeColor="@color/colorAccent" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
general hint when referring to ids please use  `@id` instead of `@+id` 

example    `app:layout_constraintTop_toBottomOf="@id/button_barrier"`